homeassistant/components/media_player/sonos.py

```diff
@@ -42,6 +42,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     players = None
     hosts = config.get('hosts', None)
+    group_devices = config.get('group_devices', False)
```

Comment:
maybe `coordinators_only` as a non-grouped player is it's own coordinator?

I would almost lean to hide non coordinators by default and make that the option to enable.

@SEJeff, what do you think?